src/modules/dom.js

```diff
@@ -0,0 +1,37 @@
+import { fetchGames } from './gameAPI.js';
+
+// Function responsible for generating a single card
+
+const generateCard = (
+  singleObj,
+) => `<div class="card" id="card_${singleObj.id}">
+  <img src="${singleObj.thumbnail}" alt="" class="card__img">
+  <div class="card__desc">
+      <p>${singleObj.title}</p>
+      <i class="fa-regular fa-heart"></i>
+  </div>
+  <span class="card__likes">5 likes</span>
+  <button class="card__btn">Comments</button>
+  <button class="card__btn">Reservations</button>
+</div>`;
+
```

Comment:
Good job having the markup HTML injected by the DOM, however, I think it would be better to have this markup as a module and imported to your `dom.js` file